### PR TITLE
fix(overhead/aircond): make aircond knobs continuous

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

* Create FBW_Continuous_Knob_Finite_From_Simvar template
* Use FBW_Continuous_Knob_Finite_From_Simvar in aircond knobs

## Screenshots (if necessary)

N/A

## References

IRL pilots (enough of a reference for this small behavior)

## Additional context

Discord username (if different from GitHub): someperson#4953

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/ModelBehaviorDefs/A32NX/A32NX_Interior.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    Main Parameters:
+            - ANIM_SIMVAR           <no default>        The name of the simvar to use for the knob
+            - ANIM_SIMVAR_UNITS     Default: "percent"  The simvar units
+            - ANIM_SIMVAR_MIN       Default: 0          The lower bound of the position
+            - ANIM_SIMVAR_MAX       Default: 100        The upper bound of the position
+            - INCREMENT             Default: 1          The increment for each click / hold update
+
+    Authors: Benjamin Dupont (@Benjozork) 10/10/20
+    -->
+    <Template Name="FBW_Continuous_Knob_Finite_From_Simvar">
```
Comment (lukecologne) [inline A32NX/ModelBehaviorDefs/A32NX/A32NX_Interior.xml:15]: Why the FBW prefix? Aren't we using A32NX as a "convention"?

**Comment by Benjozork** [comment]
Needs a changelog, will add soon

File: A32NX/ModelBehaviorDefs/A32NX/A32NX_Interior.xml (hunk 1 above)
Comment (Benjozork) [reply A32NX/ModelBehaviorDefs/A32NX/A32NX_Interior.xml:15]: So that it's not tied to the A320, future-proofing basically.

**Review by lukecologne — APPROVED** [review]
LGTM, just needs the changelog

**Comment by danbw9** [comment]
Name: danbw9#7028
Date of testing: 11/10/2020
Version of the sim: 1930
PR Tested: #1238 

Testing technique: Try-out
Steps made to test the Pull Request:

Loaded the plane C&D, checked the functionality of all the knobs and they work as described. Did a normal set up and take off and no issues discovered. 

Overall Rating: Good
Conclusions of the testing: Nice work! Only thing I would mention is that the rotation of the knobs is pretty slow, but that is just being nitpicky. LGTM.

**Comment by Benjozork** [comment]
Changelog added and rebased on master.

**Comment by Benjozork** [comment]
I will merge this myself as I need it in master to start the DCDU PR, it's a simple enough change and it's been tested.

**Pull request merged**